tssc/step_implementers/create_container_image/buildah.py

```diff
@@ -67,10 +67,15 @@
     'format': 'oci'
 }
 
-REQUIRED_ARGS = {
-    # Image destination, without version
-    'destination' : None
-}
+REQUIRED_CONFIG_KEYS = [
+    'imagespecfile',
+    'context',
+    'tlsverify',
```

Comment:
Can't we just handle tlsverify by a default? I.e. default to true? Must it always be there?

Reply:
it has a default, but if people are sneaky they can pass `None` via the config and break things. so even values with defaults, if actually required, need to be marked as required.